evolve_text.py

```diff
@@ -1,20 +1,20 @@
-"""
-Evolutionary algorithm, attempts to evolve a given message string.
+"""FILE HEADER
+    Evolutionary algorithm, attempts to evolve a given message string.
 
-Uses the DEAP (Distributed Evolutionary Algorithms in Python) framework,
-http://deap.readthedocs.org
+    Uses the DEAP (Distributed Evolutionary Algorithms in Python) framework,
+    http://deap.readthedocs.org
 
-Usage:
-    python evolve_text.py [goal_message]
+    Usage:
```

Comment:
Does this mean you indented it with spaces or tabs? I never get this right. Use spaces. 👍